packages/server/src/api/routes/tests/couchTestUtils.js

```diff
@@ -118,15 +118,15 @@ exports.clearApplications = async request => {
 exports.createUser = async (
   request,
   appId,
-  username = "babs",
+  email = "babs",
```

Comment:
This isn't a valid email address by default, probably should have default functionality of it be that it works not fails.